packages/spacecat-shared-data-access/docs/schema.json

```diff
@@ -430,6 +430,89 @@
           }
         }
       }
+    },
+    {
+      "TableName": "spacecat-services-site-candidates",
+      "KeyAttributes": {
+        "PartitionKey": {
+          "AttributeName": "baseUrl",
+          "AttributeType": "S"
+        }
+      },
+      "NonKeyAttributes": [
```

Comment:
what would be the benefit of associating the site-candidates with the actual site entities?

Reply:
it would allow to reliably identify that a given site was added via site detection versus manual. this might be interesting stats/reporting in the future, and easy to do...

Reply:
added the `siteId` field to the `site-candidate` model